scenes/MainScene.tscn

```diff
@@ -14,6 +15,8 @@ tonemap_mode = 2
 [node name="GameWorld" type="Spatial"]
 script = ExtResource( 1 )
 
+[node name="Root" parent="." instance=ExtResource( 4 )]
+
 [node name="Sun" type="DirectionalLight" parent="."]
 transform = Transform( 1, 0, 0, 0, 0.707107, 0.707107, 0, -0.707107, 0.707107, 0, 14, 0 )
```

Comment:
So basically the sun is a giant directional light that you point in a particular direction to get a particular brightness?